packages/block-editor/package.json

```diff
@@ -24,6 +24,7 @@
 	"dependencies": {
 		"@babel/runtime": "^7.9.2",
 		"@wordpress/a11y": "file:../a11y",
+		"@wordpress/api-fetch": "file:../api-fetch",
```

Comment:
the block-editor is a generic JS package not dependent of any API or backend, it shouldn't have a dependency to api-fetch or WordPress in any way. I'm not sure I know a lot about the current issue being fixed here but I though I'd mention this.

Reply:
Ah that makes sense, thank you! I moved it to `@wordpress/editor` for now, but now `@wordpress/edit-navigation` depends on editor just for this one function. I wonder what would be the best place to import this from in both editor AND edit-navigation. I've been thinking about considering the two as disjoint use cases and just copying&pasting that function, but the ultimate expectation is to have the suggestion list work the same in both the editor and in nav-menus.php so I'd rather stick to imports.

Reply:
I had a look through the packages and yeah, I'm not sure there is a good place. And my feedback here will probably leave you with more questions than answers.

The main problem with importing from `editor` is that I think the unused store for the editor package will be initialized as a side-effect of importing from the package:
https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/store/index.js#L34

I don't think it's a massive issue, but that might be an argument towards duplicating.

I did wonder if `core-data` could be an option (it's already a dependency of `edit-navigation` via other packages), though `core-data` is very much built around state management as its central purpose, so this might entail changing it to a selector/resolver and storing the suggestions in the reduxy store. 

Not sure whether there's a reason that it isn't implemented in that way to begin with (perhaps related to the caching behavior in `core-data`, or could just be an artifact of the chronology things were developed in).

All-in-all, I would probably duplicate the function and leave helpful comments above both functions pointing to one another and maybe consider adding unit tests.

Reply:
@talldan I just updated this PR to use a copy of that function. Travis fails because of some package-lock stuff that I will fix later today - your review is highly appreciated!